portal/component-tests/login/weve-sent-you-another-link.component-test.js

```diff
@@ -0,0 +1,16 @@
+const pageRenderer = require('../pageRenderer');
```

Comment:
The linter here is failing due to the file name:
https://github.com/UK-Export-Finance/dtfs2/actions/runs/6744341189/job/18334007250?pr=2268